app/experimenter/experiments/tests/api/v5/test_serializers/test_nimbus_experiment_serializer.py

```diff
@@ -968,7 +968,7 @@ def test_cant_review_for_requesting_user(self):
         self.assertFalse(serializer.is_valid(), serializer.errors)
         self.assertIn("publish_status", serializer.errors)
 
-    def test_can_review_for_requesting_user_when_idle(self):
```

Comment:
Was this added so that we can manually approve? Should this still be allowed?

Reply:
I am not sure about this 🤔

Reply:
if they still uses manually approve thing

Reply:
Oh interesting yeah I don't think that workflow exists anymore, I don't see any reason to keep that.